telemetry/definitions/commonDefinitions.json

```diff
@@ -3476,9 +3553,58 @@
                 }
             ]
         },
+        {
+            "name": "codeTransform_initiateTransform",
+            "description": "User initiates code transform from Q Chat Prompt.",
+            "metadata" : [
+                {
+                    "type": "codeTransformSessionId",
+                    "required": true
+                },
+                {
+                    "type": "result",
+                    "required": true
+                },
+                {
+                    "type": "reason",
+                    "required": false
+                }
+            ]
+        },
+        {
+            "name": "codeTransform_invokeApi",
```

Comment:
Please plan to deduplicate this metric in the clients. Emitting a metric for every service call is very noisy.

Reply:
Would you recommend emitting API invocation metric only when the API call fails? The RequestId emitted in these API metrics have helped us find corresponding logs on code-whisperer side.